cli/index.ts

```diff
@@ -333,6 +333,13 @@ async function cli(args: CommandLineArgs): Promise<Options> {
     ),
   );
 
+  console.log(
```

Comment:
Adding a warning here so that people will not attempt to use file storage and CSV export yet (although they're pretty much no-ops anyways for now)

Previously we opted not to publish the CLI changes altogether until file storage and CSV are fully in, but this will result in inconsistencies between the prod CLI and the release branch that will be continuously updated (e.g. prod CLI does not know about some new files and will not delete them, leaving extraneous files in the generated starter code)